vllm/entrypoints/openai/serving_tokenization.py

```diff
@@ -65,10 +65,10 @@ async def create_tokenize(
         if isinstance(request, TokenizeChatRequest):
             model_config = self.model_config
 
-            conversation, mm_futures = parse_chat_messages(
+            conversation, mm_future = parse_chat_messages(
                 request.messages, model_config, tokenizer)
 
-            if mm_futures:
+            if mm_future:
```

Comment:
Let's name it `mm_data_future` to be consistent with the change in `serving_chat.py`